docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -9,45 +9,56 @@ description: Hardware Key Support
   Hardware Key Support requires Teleport Enterprise.
 </Admonition>
 
-By default, `tsh`, Teleport Connect, and other Teleport clients store a user's login session directly on their filesystem. If a user's filesystem is compromised, any of their active Teleport login sessions would also be compromised.
+By default, `tsh`, Teleport Connect, and other Teleport clients store a user's login session directly on the local filesystem. 
```

Comment:
It's not really the "session" that's stored on the filesystem, but the certificates. Should we be specific?